s3/src/main/scala/org/apache/pekko/stream/connectors/s3/javadsl/S3.scala

```diff
@@ -485,8 +486,11 @@ object S3 {
    * @return A [[pekko.stream.javadsl.Source]] containing the objects data as a [[pekko.util.ByteString]] along with a materialized value containing the
    *         [[pekko.stream.connectors.s3.ObjectMetadata]]
    */
-  def getObject(bucket: String, key: String): Source[ByteString, CompletionStage[ObjectMetadata]] =
-    new Source(S3Stream.getObject(S3Location(bucket, key), None, None, S3Headers.empty).toCompletionStage())
+  def getObject(bucket: String, key: String): Source[ByteString, CompletionStage[ObjectMetadata]] = {
+    val objectSource: SourceToCompletionStage[ByteString, ObjectMetadata] =
```

Comment:
I think this type can be simplified but as before will have to check out locally (also applies for other future instances).